api/include/opentelemetry/baggage/propagation/baggage_propagator.h

```diff
@@ -46,6 +46,15 @@ class BaggagePropagator : public context::propagation::TextMapPropagator
     auto baggage                   = Baggage::FromHeader(baggage_str);
     return SetBaggage(context, baggage);
   }
+
+  bool Fields(nostd::function_ref<bool(nostd::string_view)> callback) const noexcept override
+  {
+    if (callback(kBaggageHeader))
```

Comment:
`return callback(kBaggageHeader)`?

Reply:
Yes, this looks better. Will change the code before merging.

Reply:
It's done now.